CMakeLists.txt

```diff
@@ -187,6 +187,18 @@ if (APPLE)
   set(OLDEST_OSX_SUPPPORTED "11.7")
 elseif (UNIX)
   set(LINUX 1)
+  find_program(LDD ldd DOC "util for examing shared object dependency")
```

Comment:
Please add a comment "Identify whether building with (and targeting) musl libc".